packages/app-project/src/components/AuthModal/shared/components/withCustomFormik/withCustomFormik.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Formik } from 'formik'
+
+function withCustomFormik (WrappedComponent) {
+  class FormikHOC extends React.Component {
+    constructor () {
+      super()
+
+      this.handleChangeWithCallback = this.handleChangeWithCallback.bind(this)
+    }
+
+    handleChangeWithCallback (event, formikProps) {
+      formikProps.handleChange(event)
+      this.props.onChange(event, formikProps)
+    }
+
+    render () {
+      const { forwardedRef, initialValues, onSubmit, ...rest } = this.props
+      return (
+        <Formik
+          initialValues={initialValues}
+          onSubmit={onSubmit}
+          {...rest}
+        >
+          {(innerProps) => {
+            const {
+              handleChange,
+              ...restOfInnerProps
```

Comment:
`innerProps.handleBlur` will be passed down here, so filter it out here if you really do want to remove support for it.

Reply:
I'm not sure if I want to do that because I don't understand why it's broken yet.